Products/ZenHub/tests/test_interceptors.py

```diff
@@ -47,9 +60,18 @@ def setUp(self):
         self.zenhub.deferToWorker.return_value = 'deferred'
         self.wi = WorkerInterceptor(zenhub=self.zenhub, service=self.service)
         self.broker = Mock(name='pb.Broker', spec_set=pb.Broker)
+        self.broker.unserialize = Mock()
```

Comment:
this will work better as:
and unserialize will be included in the spec
```suggestion
        self.broker = Mock(pb.Broker, name='pb.Broker')
```